R/bcmaps-package.R

```diff
@@ -0,0 +1,23 @@
+# Copyright 2023 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+
+
+#' @keywords internal
+"_PACKAGE"
+
+#' @import sf
+#' @importFrom utils download.file unzip
+
+## usethis namespace: start
+#' @importFrom lifecycle deprecated
```

Comment:
are we actually using this function anywhere? Sorry if I missed it.

Reply:
Yes, we are using it for all the deprecation pieces.